cypress.config.ts

```diff
@@ -5,35 +5,41 @@ import * as dotenv from "dotenv";
 
 dotenv.config({ path: "./.env.local" });
 
+const setupNodeEvents = (
+    on: Cypress.PluginEvents,
+    config: Cypress.PluginConfigOptions
+): Cypress.PluginConfigOptions => {
+    // implement node event listeners here
+    registerCodeCoverageTasks(on, config);
+
+    on("task", {
+        table(message: any[]): null {
+            if (message.length > 0) {
+                console.table(message);
+            }
+            return null;
```

Comment:
As mentioned below, are these your changes?

Reply:
Newest VFB-16 iteration has this gone - will fix all when it is merged into main